docs/project-charter.md

```diff
@@ -0,0 +1,196 @@
+# LoopBack Charter
+
+_note: the purpose of a project charter is to provide a brief introduction_
```

Comment:
i assume the text in italic (i.e. note, directions, ex) will be removed eventually and it's there now just to help with the review?

Reply:
Yes, these would be removed before landing the PR. I've left them there to make it easier to review.